### ISSUE TITLE
Animations skip when data updated faster than animation completion

### ISSUE THREAD

**Issue by mattoni** [recharts/recharts#1431]
## Do you want to request a feature or report a bug?
bug

## What is the current behavior?
Animations are interrupted if data is changed too quickly

## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: http://jsfiddle.net/ndLhnegs/).

(jsfiddle template is broken)

Change the data of a component faster than the animation finishes. When the new data arrives the animation "jumps" to complete then starts the next one.

## What is the expected behavior?
The current animation will complete before starting the next.

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?

Recharts 1.1.0
Firefox 61.0.1

**Comment by xile611** [comment]
Sorry for not handling your issue in time, please try the latest version. If the problem persists, please open a new issue according to the issue guide.
